add-tourpackage.php

```diff
@@ -0,0 +1,483 @@
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Price</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="w-100" id="price">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Header Text</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="w-100" id="h-text">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Description</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <textarea name="" id="description" rows="15" class="w-100"></textarea>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Select the Destinations</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <div class="row">
+                                    <div class="col-12">
+                                        <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal">
+                                            Click here to select destinations
+                                        </button>
+                                    </div>
+                                    <div class="col-12">
+                                        <span id="desIDList"></span>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Number of Vehicles</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="number" class="w-100" id="no-of-vehicles">
```

Comment:
Add `min`, `max` attributes to the input element.

Reply:
Ok